documentation/command/baseline.md

```diff
@@ -13,6 +13,12 @@ Baseline is for introducing Flyway to [existing databases](/documentation/learnm
 at a specific version. This will cause [Migrate](/documentation/command/migrate) to ignore all migrations
 up to and including the baseline version. Newer migrations will then be applied as usual.
 
+## Resetting the baseline
+
+When you have many migrations, it might be desirable to reset your baseline migration. This will allow you to reduce the build of old scripts.
```

Comment:
It might be worth rephrasing `This will allow you to reduce the build of old scripts`

It doesn't immediately signal any value imo, and it's hard to figure out what it actually means